api/restHandler/app/BuildPipelineRestHandler.go

```diff
@@ -1053,6 +1055,66 @@ func (handler PipelineConfigRestHandlerImpl) GetCIPipelineById(w http.ResponseWr
 	common.WriteJsonResp(w, err, ciPipeline, http.StatusOK)
 }
 
+func (handler PipelineConfigRestHandlerImpl) GetCIPipelineByPipelineId(w http.ResponseWriter, r *http.Request) {
+	token := r.Header.Get("token")
+	var ciPipelineId int
+	var err error
+	v := r.URL.Query()
+	pipelineId := v.Get("pipelineId")
+	if len(pipelineId) != 0 {
```

Comment:
what about zero length pipelineId ??

Reply:
okay, this case was not handled, have handled it now